### ISSUE TITLE
PADDLE2ONNX: cannot convert  SFNet-R18 into onnx 

### ISSUE THREAD

**Issue by zlin-monarch** [PaddlePaddle/PaddleSeg#2973]
### 问题确认 Search before asking

- [X] 我已经查询[历史issue](https://github.com/PaddlePaddle/PaddleSeg/issues)(包括open与closed)，没有发现相似的bug。I have searched the [open and closed issues](https://github.com/PaddlePaddle/PaddleSeg/issues) and found no similar bug report.

### Bug描述 Describe the Bug

When converting sfnet_resnet50_os8 from static graph into onnx model. I got this error. Hope someone can help solve it! 
```
raise Exception(
Exception: Error happened when mapping node ['grid_sampler_0'] to onnx, which op_type is 'grid_sampler' with inputs: {'Grid': ['tmp_2'], 'X': ['relu_55.tmp_0']} and outputs: {'Output': ['grid_sample_0.tmp_0']}, specific error: Error happened when mapping node ['fill_any_like_0'] to onnx, which op_type is 'fill_any_like' with inputs: {'X': ['grid_sampler3_grid_sampler3_fill_constant_5.tmp_0']} and outputs: {'Out': ['grid_sampler3_grid_sampler3_full_like_0.tmp_0']}, specific error: array(0) has type numpy.ndarray, but expected one of: int
```
- Converting command 
```
paddle2onnx  --model_dir args.save_dir \
        --model_filename model.pdmodel \
        --params_filename model.pdiparams \
        --save_file model.onnx \
        --opset_version 11
```

- Input size: (1, 3, 1008, 1008) 
- Model is from 
```
cfg = Config('configs/sfnet/sfnet_resnet50_os8_cityscapes_1024x1024_80k.yml')
cfg.check_sync_info()
model = cfg.model
model_path = model.pdparams
paddle.save(model.state_dict(), model_path) 

os.system(f'{Paddleseg}/export.py --config configs/sfnet/sfnet_resnet50_os8_cityscapes_1024x1024_80k.yml \
--model_path {model_path} \
--input_shape 1 3 1008 1008 \
--save_dir {args.save_dir} \
--model_name {args.model_name}')
```

- Platform information 
```
JetsonAGX Orin (Jetpack 5.0) 
Cuda 11.4
Paddle2onnx 1.0.5
Paddlepaddle 2.4 
paddleseg release/2.7 
onnx 1.11.0
python 3.8
```

### 复现环境 Environment

- Jetson AGX Orin (aarch64): jetpack 5.0 
- CUDA: 11.4
- Paddle2onnx 0.9.2
- Paddlepaddle 2.4 
- paddleseg release/2.7 
- onnx 1.11.0

### Bug描述确认 Bug description confirmation

- [X] 我确认已经提供了Bug复现步骤、代码改动说明、以及环境信息，确认问题是可以复现的。I confirm that the bug replication steps, code change instructions, and environment information have been provided, and the problem can be reproduced.

### 是否愿意提交PR？ Are you willing to submit a PR?

- [x] 我愿意提交PR！I'd like to help by submitting a PR!

**Comment by jiangjiajun** [comment]
Update Paddle2ONNX to latest version `v1.0.5` will solve this problem.

**Comment by zlin-monarch** [comment]
Hi, I upgrade the `paddle2onnx` into version 1.0.5 but the issue is still here :(
This is steps for installing version 1.0.5
```
git clone https://github.com/PaddlePaddle/Paddle2ONNX.git
cd Paddle2ONNX
python3 setup.py install
```

**Comment by zlin-monarch** [comment]
Hi, is there any updates?

**Comment by jiangjiajun** [comment]
paddle2onnx 1.0.5 shouldn't throw error like this `raise Exception(
Exception: Error happened when mapping node ['grid_sampler_0'] to onnx, which op_type is 'grid_sampler' with inputs: {'Grid': ['tmp_2'], 'X': ['relu_55.tmp_0']} and outputs: {'Output': ['grid_sample_0.tmp_0']}, specific error: Error happened when mapping node ['fill_any_like_0'] to onnx, which op_type is 'fill_any_like' with inputs: {'X': ['grid_sampler3_grid_sampler3_fill_constant_5.tmp_0']} and outputs: {'Out': ['grid_sampler3_grid_sampler3_full_like_0.tmp_0']}, specific error: array(0) has type numpy.ndarray, but expected one of: int`

The old version of Paddle2ONNX is implemented by Python, but 1.0.5 Paddle2ONNX is implemented by C++. Please make sure you are installed paddle2onnx 1.0.5
